extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -334,6 +336,10 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
    */
   schedulePage_() {
     const page = this.createAdPage_();
+    if (!page) {
+      // Creation of ad has failed.
```

Comment:
Should we throw an error or warning here? 🤔 

It may be a bit noisy though if we did.

Reply:
I think this should only happen in the fake case which will already user().warn()

Reply:
Sounds good to me! 😄